AnnotationTool/myapp/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path("", views.home, name="home"),
+    path("upload_image/", views.upload_image_page, name="upload_image_page"),
+    path("test/", views.get_data, name="test")
```

Comment:
You can remove the `test/` path that I put with you or change it into something more meaningful.